cabal.project

```diff
@@ -77,3 +69,13 @@ allow-newer: servant-multipart-api:servant
 
 -- ghc 9
 allow-newer: tdigest:base
+
+-- ghc 9.2
+allow-newer: basement:base
+allow-newer: memory:base
+allow-newer: openssl-streams:bytestring
+allow-newer: servant-multipart:bytestring
+allow-newer: servant-multipart:bytestring
+allow-newer: servant-multipart:http-media
+allow-newer: servant-multipart:lens
```

Comment:
* https://github.com/haskell-servant/servant-multipart/pull/59

Reply:
@brandon-leapyear These ones for servant-multipart and the one for openssl-streams can be removed now. If you rebase, the diff will also get smaller now because allow-newers were removed on master.